packages/editor/src/client/routes/articleEditor.tsx

```diff
@@ -379,7 +379,7 @@ export function ArticleEditor({id}: ArticleEditorProps) {
 
       setChanged(false)
 
-      publishData === new Date() || publishDate < new Date()
+      publishDate <= new Date()
```

Comment:
just a little suggestion since the translation key is the only thing changing between the if and else you could also write this like:
```
Notification.success({
  title: t(publishDate <= new Date() ? 'articleEditor.overview.articlePublished' : 'articleEditor.overview.articlePending'),
  duration: 2000
})
```
This way it's a bit more clear that only the message changes